go.mod

```diff
@@ -6,15 +6,16 @@ replace k8s.io/client-go => k8s.io/client-go v0.20.0
 
 require (
 	github.com/evanphx/json-patch v4.9.0+incompatible
-	github.com/go-logr/logr v0.3.0
+	github.com/go-logr/logr v0.3.0 // indirect
 	github.com/rancher/eks-operator v1.0.6-rc1
 	github.com/rancher/fleet/pkg/apis v0.0.0-20210225010648-40ee92df4aea
-	github.com/rancher/lasso v0.0.0-20200905045615-7fcb07d6a20b
-	github.com/rancher/norman v0.0.0-20210219183327-731b8482505c
-	github.com/rancher/rancher/pkg/apis v0.0.0-20210222182625-a85f4d1f87fe
-	github.com/rancher/rancher/pkg/client v0.0.0-20210222182625-a85f4d1f87fe
-	github.com/rancher/rke v1.2.6-rc5
-	github.com/rancher/wrangler v0.7.3-0.20201028210318-d73835950c29
+	github.com/rancher/gke-operator v1.0.1-rc6
```

Comment:
Why are we only adding GKE operator back and not updating the others like EKS ? Curious what the actual problem was that requires us to bring it back, as it would be ideal to fix that instead.

Reply:
Oh, I didn't catch this. We shouldn't need this dependency. Updating `github.com/rancher/pkg/apis` should be enough.

Reply:
Meant to undo that, fixed now